### DISCUSSION TITLE
UPnP / Airplay / Chromecast proxy

### DISCUSSION THREAD

**Discussion by garfieldairlines** [music-assistant/support#2920] [Feature requests and ideas]
My request is pretty dumb but it would help me a lot.
Basically allow for Music Assistant to be a "Line in" and to feed it music via UPnP / Airplay / Chromecast from my smartphone (for example) and configure in Music Assistant on which speakers I want to send the output.

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/1757

Airplay and Spotify Connect are moving up the devs priority list

**Reply by garfieldairlines** [reply]
Ah thanks sorry, I was looking left and right and haven't found this one…
